dispatch/route.go

```diff
@@ -122,7 +136,8 @@ func NewRoutes(croutes []*config.Route, parent *Route) []*Route {
 // Match does a depth-first left-to-right search through the route tree
 // and returns the matching routing nodes.
 func (r *Route) Match(lset model.LabelSet) []*Route {
-	if !r.Matchers.Match(lset) {
+
```

Comment:
Again, I'd prefer to save whitespace and not have this blank line here, which doesn't really help to understand the structure of the method. (Note that there was no blank line before).